xmeffects.js

```diff
@@ -287,3 +293,4 @@ var effects_t1 = [  // effect functions on tick 1+
   eff_unimplemented,  // y
   eff_unimplemented   // z
 ];
+})(window || {});
```

Comment:
Ok. Let's be consistent between xm.js and xmeffects.js then. I think in a unit test setting we'd just override window anyway, right?